scenario-compiler/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala

```diff
@@ -1663,6 +1663,42 @@ class SpelExpressionSpec extends AnyFunSuite with Matchers with ValidatedValuesD
     }
   }
 
+  test("should convert list of map entries into map") {
+    // this is a special case
```

Comment:
I would remove these comments because it's says only that we didn't handle sth in the past and now test proves that we handle this situation